pkg/config/config_test.go

```diff
@@ -39,12 +41,26 @@ const (
 )
 
 func TestNewConfigPermissionDenied(t *testing.T) {
-	expectedError := errors.New("open /tmp.yaml: permission denied")
-
 	_, err := config.NewConfig(appName, "../../../../../../../../../../../../tmp")
```

Comment:
the test worked on a local windows env. testing with more subpaths here 

```suggestion
	_, err := config.NewConfig(appName, "../../../../../../../../../../../../../../../../../tmp")
```